src/main/java/org/jspecify/nullness/NullUnmarked.java

```diff
@@ -49,10 +49,9 @@
  * <h2>Null-marked and null-unmarked code</h2>
  *
  * <p>{@link NullMarked} and this annotation work as a pair to include and exclude sections of code
- * from null-marked status.
- *
- * <p>Code is considered null-marked if the most narrowly enclosing element annotated with either of
- * these two annotations is annotated with {@code @NullMarked}.
+ * from null-marked status (respectively). Specifically, code is considered null-marked if the most
+ * narrowly enclosing element annotated with either of these two annotations is annotated with
+ * {@code @NullMarked}.
```

Comment:
I find this sentence vague and confusing. What about:

```
Specifically, code is considered *null-marked* if the most narrowly enclosing element is annotated with `@NullMarked`, otherwise, it is considered *null-unmarked*.
```

This would require moving the first sentence of the next paragraph into this sentence.

Reply:
(For starters, I made the current wording slightly more accurate and *more* awkward.)

I think the reword doesn't mean the same thing. The most narrowly enclosing element might not be annotated. Also the "code" we're talking about could be a method signature itself, which might be annotated `@NullUnmarked`.

Frankly, trying to talk about "code" might just not be working.

Here's a totally different approach:

"The signature and body of a `@NullMarked` element is considered null-marked code, and the signature and body of a `@NullUnmarked` element is considered null-unmarked code, where more narrowly enclosing elements have precedence over those at a broader scope like a module. Finally, the signature and body of an element having neither annotation nor being enclosed by any element having either annotation is considered null-unmarked."

Or we could write it more algorithmically, saying "to determine whether code is null-marked, <take these steps>".

it's crazy how much harder this is to write than to understand. It's just inclusion/exclusion.

Reply:
I've left a TODO comment about improving this, and am gonna go ahead and merge this now -- still happy to make more changes tomorrow.